src/jshal.js

```diff
@@ -215,4 +215,31 @@ mergeInto(LibraryManager.library, {
   mp_js_hal_audio_is_expression_active: function () {
     return board.audio.isSoundExpressionActive();
   },
+
+  mp_js_hal_log_delete: function (full_erase) {
+    board.dataLogging.delete();
+  },
+
+  mp_js_hal_log_set_mirroring: function (serial) {
+    board.dataLogging.delete();
```

Comment:
Is this right?

Reply:
Nope. [`26e9c8b`](https://github.com/microbit-foundation/micropython-microbit-v2-simulator/pull/36/commits/26e9c8b1d85e7dd05ae5ddc100b50d09d1e7c93b).